wifi_manager/wifi_manager.py

```diff
@@ -240,28 +245,45 @@ def start_config(self) -> None:
         gc.collect()
         self.logger.debug('Goodbye from WiFiManager')
 
+    def add_app_route(self,
```

Comment:
use backwards compatible function name 
```suggestion
    def add_url_rule(self,
```